openpower/configs/hostboot/zaius.config

```diff
@@ -0,0 +1,67 @@
+# The Serial Flash Controller is the AST2500 BMC.
+set   SFC_IS_AST2500
+unset SFC_IS_AST2400
+unset BMC_DOES_SFC_INIT
+unset SFC_IS_IBM_DPSS
+set   ALLOW_MICRON_PNOR
+set   ALLOW_MACRONIX_PNOR
+
+# VPD options.
+set MVPD_READ_FROM_HW
+set MVPD_WRITE_TO_HW
+set MVPD_READ_FROM_PNOR
+set MVPD_WRITE_TO_PNOR
+unset DJVPD_READ_FROM_HW
```

Comment:
Yes it doesn't work yet as the necessary attributes are not present in zaius-xml yet.